packages/graphql/tests/tck/tck-test-files/interface-relationships/update/create.test.ts

```diff
@@ -89,29 +89,29 @@ describe("Interface Relationships - Update create", () => {
         });
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
-            "MATCH (this:\`Actor\`)
-            CREATE (this_create_actedIn_Movie0_node_Movie:Movie)
-            SET this_create_actedIn_Movie0_node_Movie.title = $this_create_actedIn_Movie0_node_Movie_title
-            SET this_create_actedIn_Movie0_node_Movie.runtime = $this_create_actedIn_Movie0_node_Movie_runtime
-            MERGE (this)-[this_create_actedIn_Movie0_relationship:ACTED_IN]->(this_create_actedIn_Movie0_node_Movie)
-            SET this_create_actedIn_Movie0_relationship.screenTime = $this_create_actedIn_Movie0_relationship_screenTime
-            WITH *
-            WITH this
-            CALL {
-            WITH this
-            CALL {
-                WITH this
-                MATCH (this)-[:ACTED_IN]->(this_Movie:Movie)
-                RETURN { __resolveType: \\"Movie\\", runtime: this_Movie.runtime, title: this_Movie.title } AS actedIn
-                UNION
-                WITH this
-                MATCH (this)-[:ACTED_IN]->(this_Series:Series)
-                RETURN { __resolveType: \\"Series\\", episodes: this_Series.episodes, title: this_Series.title } AS actedIn
-            }
-            RETURN collect(actedIn) AS actedIn
-            }
-            RETURN collect(DISTINCT this { .name, actedIn: actedIn }) AS data"
-        `);
+"MATCH (this:\`Actor\`)
+CREATE (this_create_actedIn_Movie0_node_Movie:Movie)
+SET this_create_actedIn_Movie0_node_Movie.title = $this_create_actedIn_Movie0_node_Movie_title
+SET this_create_actedIn_Movie0_node_Movie.runtime = $this_create_actedIn_Movie0_node_Movie_runtime
+MERGE (this)-[this_create_actedIn_Movie0_relationship:ACTED_IN]->(this_create_actedIn_Movie0_node_Movie)
+SET this_create_actedIn_Movie0_relationship.screenTime = $this_create_actedIn_Movie0_relationship_screenTime
+WITH *
+WITH *
```

Comment:
Double `WITH *`

Reply:
There is not much we can do about this until we fully refactor this I'm afraid :confused:

Reply:
Sure!